src/modules/ROOT/pages/changelog/202309/00.json

```diff
@@ -0,0 +1,261 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1484"
+      ]
+    },
+    {
+      "what": [
+        "an icon containing an exclamation mark to a page block toolbar indicating that the concrete page block is used in another page layout."
+      ],
+      "why": [
+        "to mark a page block that is already used in another page layout."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1495"
+      ]
+    },
+    {
+      "what": [
+        "the option to change the default colors of {PRODUCT_NAME} applications in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide the user with more freedom to style {PRODUCT_NAME}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1469"
+      ]
+    },
+    {
+      "what": [
+        "a rounded circle icon that marks a configuration option as performance impacting."
+      ],
+      "why": [
+        "to mark which features would impact the performance of an application."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1490"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "triggering of unsaved changes message after user wants to delete a modified resource like a page, namespace, module or chart."
+      ],
+      "why": [
+        "to provide a more intuitive way to navigate in {APP_NAME_COMPOSE} after trying to delete a resource."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1466"
+      ]
+    },
+    {
+      "what": [
+        "the navigation flow when switching between different {APP_NAME_COMPOSE} resources using the sidebar navigation menu.",
+        "For example, when trying to switch from one page's builder mode to another, you were redirected to the edit mode of the page."
+      ],
+      "why": [
+        "to provide a more intuitive way to navigate between {APP_NAME_COMPOSE} resources."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1457"
+      ]
+    },
+    {
+      "what": [
+        "the functionality of the delete and un-delete button by extending it to show a spinner while a resource is being deleted or un-deleted."
+      ],
+      "why": [
+        "to show the user the current state of the delete or un-delete process."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1464"
+      ]
+    },
+    {
+      "what": [
+        "the syntax of forms for creating and modifying a resource to better support mobile users in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide a better UI and UE for mobile users."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1253"
+      ]
+    },
+    {
+      "what": [
+        "the updating cycle of page blocks showing the same data to refresh after a change has been made to that data in a different page block."
```

Comment:
This sounds confusing.
Under the hood, we're keeping all page blocks in sync if they're using the same underlying data source.

So perhaps something along the lines of 

"Changed page blocks to refresh in case the underlying data was modified by a different page block on the same page".